.github/workflows/go-test.yaml

```diff
@@ -131,6 +131,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - name: setup golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.20"
```

Comment:
Did you want to keep this step with the one above on purpose?

Reply:
no, space was missed added now